addons/disable-sprite-wobble/addon.json

```diff
@@ -9,17 +9,53 @@
     {
       "name": "CST1229",
       "link": "https://github.com/cst1229/"
+    },
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
     }
   ],
   "userstyles": [
     {
       "url": "userstyle.css",
       "matches": ["projects"]
+    },
+    {
+      "url": "disable-wobble.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": {
+          "mode": "no-wobble"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "id": "mode",
+      "name": "Mode",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "none",
+          "name": "Fix bug"
+        },
+        {
+          "id": "no-wobble",
+          "name": "Fix bug and disable sprite wobble"
```

Comment:
I would like if it was clearer why this setting was here. With the update notification when this is initially released, this will make sense, but after that, it wouldn't fit here.

<details><summary>One idea...</summary>
<p>

```suggestion
          "name": "Fix bug and disable sprite wobble (original)"
```

</p>
</details>

We could also think about eventually moving either this or the bug fix to a new addon or something.

Reply:
Maybe add an info.

Reply:
@DNin01Adding "original" to the option is also okay. Commit your suggestion if you like it.

Reply:
> @DNin01 Adding "original" to the option is also okay. Commit your suggestion if you like it.

For now, with the update notice to go along, I think the current option name is okay. Not sure what to do afterward but we'll figure something out.